src/GatewayNode.ts

```diff
@@ -72,7 +77,9 @@ class GatewayNode extends UiComponent /* implements Checkable */ {
       // this.flag.check().then(() => log.debug(this.gateway, 'Flag success')),
       this.status.check().then(() => log.debug(this.gateway, 'Status success')).then(this.onSuccessfulCheck.bind(this)),
       this.cors.check().then(() => log.debug(this.gateway, 'CORS success')).then(this.onSuccessfulCheck.bind(this)),
-      this.origin.check().then(() => log.debug(this.gateway, 'Origin success')).then(this.onSuccessfulCheck.bind(this))
+      this.origin.check().then(() => log.debug(this.gateway, 'Origin success')).then(this.onSuccessfulCheck.bind(this)),
```

Comment:
I feel this can be refactored, but will revisit this in a future PR.

Reply:
Could you file an issue for this?

Reply:
https://github.com/ipfs/public-gateway-checker/issues/311